content/resources.md

```diff
@@ -0,0 +1,60 @@
+- [_The Open Revolution: Rewrting the Rules of the Information Age_, Rufus Pollock](https://www.goodreads.com/book/show/40515943-the-open-revolution?from_search=true&qid=VHNoWdsYe6&rank=2) ([eBook here](https://openrevolution.net/))
+- [_Rebel Code: Linux and the Open Source Revolution_, Glyn Moody](https://www.goodreads.com/book/show/289947.Rebel_Code?from_search=true&qid=VHNoWdsYe6&rank=1)
+- [_Free Innovation_, Eric von Hippel](https://www.goodreads.com/book/show/33000056-free-innovation?from_search=true&qid=0jTHYQPJiB&rank=1)
+- [_Don't Be Evil: How Big Tech Betrayed Its Founding Principles — and All of Us_, Rana Foroohar](https://www.goodreads.com/book/show/46758636-don-t-be-evil?from_search=true&qid=dbEE79pyrt&rank=1)
+- ["The Complicated Legacy of Stewart Brand’s 'Whole Earth Catalog'", Anna Wiener](https://www.newyorker.com/news/letter-from-silicon-valley/the-complicated-legacy-of-stewart-brands-whole-earth-catalog)
+- ["Open Source Under Attack: How we, the OSI and others can defend it", Chris Aniszczyk, Max Sills, Michael Cheng](https://fosdem.org/2020/schedule/event/osi/)
+- ["Stewart Brand's Whole Earth Catalog, the book that changed the world", Carole Cadwalladr](https://www.theguardian.com/books/2013/may/05/stewart-brand-whole-earth-catalog)
+- ["The Whole Earth Catalog Effect", Steven Kotler](https://www.mnn.com/lifestyle/arts-culture/stories/the-whole-earth-catalog-effect)
+- [_Open Sources: Voices from the Open Source Revolution_, Bruce Perens (ed)](https://www.oreilly.com/openbook/opensources/book/perens.html)
+- ["The Social Meaning of the Personal Computer: Or, Why the Personal Computer Revolution Was No Revolution", Bryan Pfaffenberger](https://www.jstor.org/stable/3317870)
+- [_Cyberselfish: A Critical Romp through the Terribly Libertarian Culture of High Tech_, Paulina Borsook](https://www.goodreads.com/book/show/211081.Cyberselfish_A_Critical_Romp_Through_The_Terribly_Libertarian_Culture_Of_High_Tech?from_search=true&qid=ohIVlJ6frh&rank=1)
+- [_Fire in the Valley: The Making of the Personal Computer_, Michael Swaine, Paul Freiberger](https://www.goodreads.com/book/show/22633290-fire-in-the-valley)
+- ["The Meme Hustler", Evgeny Morozov](https://thebaffler.com/salvos/the-meme-hustler)
+- ["The Economics of Open Source", C J Silverio](https://2019.jsconf.eu/c-j-silverio/the-economics-of-open-source.html)
+- ["The culture war at the heart of open source", Steve Klabnik](https://words.steveklabnik.com/the-culture-war-at-the-heart-of-open-source)
+- ["What comes after 'open source'", Steve Klabnik](https://words.steveklabnik.com/what-comes-after-open-source)
+
+## Heroes
+- ["Ethical Source at WeTransfer", Bastiaan Terhorst](https://bastiaan.cc/notes/ethical-source-at-wetransfer/)
+- [Douglas Crockford's JSON license, Wikipedia](https://en.wikipedia.org/wiki/Douglas_Crockford#%22Good,_not_Evil%22)
```

Comment:
I think we should add a representative news story about Seth Vargo. Thoughts?

Reply:
Great idea!